### PR TITLE
Update README and Website for  v1.2.0 release.

### PR DESCRIPTION
**Pull request by marinsalinas**
A

### CODE REVIEW

**Comment by gabybeitler** [comment]
looks good.

File: website/docs/d/permission.html.markdown [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+* `permission_id`:  (ConflictsWith: `permission_name`) The `id` of the permission.
+* `permission_name`:  (ConflictsWith: `permission_id`) The `name` of the permission.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+
+* `metadata`: The subnet kind metadata.
```
Comment (yannickstruyf3) [inline website/docs/d/permission.html.markdown]: change subnet to permission

File: website/docs/d/permission.html.markdown [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+* `field_mode` Allow or disallow the fields mentioned.
+* `field_name_list` The list of fields.
+
+### Metadata
+
+The metadata attribute exports the following:
+
+* `last_update_time` - UTC date and time in RFC-3339 format when the project was last updated.
+* `uuid` - Project UUID.
```
Comment (yannickstruyf3) [inline website/docs/d/permission.html.markdown]: permission uuid

File: website/docs/d/permission.html.markdown [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+* `creation_time` - UTC date and time in RFC-3339 format when the project was created.
```
Comment (yannickstruyf3) [inline website/docs/d/permission.html.markdown]: permission creation date

File: website/docs/d/permission.html.markdown [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+* `spec_version` - Version number of the latest spec.
+* `spec_hash` - Hash of the spec. This will be returned from server.
+* `name` - Project name.
```
Comment (yannickstruyf3) [inline website/docs/d/permission.html.markdown]: permission name

**Review by gabybeitler — APPROVED** [review]

**Comment by coderGo93** [comment]
looking good so far, in website there's a file for resource and datasource `volume_group`, is that correct? because it's not in any resource/datasource unless is in another branch

**Review by yannickstruyf3 — CHANGES_REQUESTED** [review]
Some copy-paste mistakes in permission docs (still referring to projects)

**Comment by yannickstruyf3** [comment]
> looking good so far, in website there's a file for resource and datasource `volume_group`, is that correct? because it's not in any resource/datasource unless is in another branch

Volume_groups are not supported. Docs should be removed

File: website/docs/d/permission.html.markdown [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+The field attribute exports the following:
+
+* `field_mode` Allow or disallow the fields mentioned.
+* `field_name_list` The list of fields.
+
+### Metadata
+
+The metadata attribute exports the following:
+
+* `last_update_time` - UTC date and time in RFC-3339 format when the project was last updated.
```
Comment (yannickstruyf3) [inline website/docs/d/permission.html.markdown]: last_update_time refers to project

File: website/docs/d/permissions.html.markdown [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+The following attributes are exported:
+
+* `api_version`: version of the API
+* `entities`: List of Permissions
+
+# Entities
+
+The entities attribute element contains the following attributes:
+
+* `metadata`: The subnet kind metadata.
```
Comment (yannickstruyf3) [inline website/docs/d/permissions.html.markdown]: permissions instead of subnet

File: website/docs/d/permissions.html.markdown [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+The field attribute exports the following:
+
+* `field_mode` Allow or disallow the fields mentioned.
+* `field_name_list` The list of fields.
+
+### Metadata
+
+The metadata attribute exports the following:
+
+* `last_update_time` - UTC date and time in RFC-3339 format when the project was last updated.
```
Comment (yannickstruyf3) [inline website/docs/d/permissions.html.markdown]: permission instead of project

File: website/docs/d/permissions.html.markdown [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+* `uuid` - Project UUID.
```
Comment (yannickstruyf3) [inline website/docs/d/permissions.html.markdown]: permission instead of project

File: website/docs/d/permissions.html.markdown [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+* `creation_time` - UTC date and time in RFC-3339 format when the project was created.
```
Comment (yannickstruyf3) [inline website/docs/d/permissions.html.markdown]: permission instead of project

File: website/docs/d/permissions.html.markdown [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+* `spec_version` - Version number of the latest spec.
+* `spec_hash` - Hash of the spec. This will be returned from server.
+* `name` - Project name.
```
Comment (yannickstruyf3) [inline website/docs/d/permissions.html.markdown]: permission instead of project

File: website/docs/d/subnets.html.markdown [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+* `vswitch_name`: The name of the vswitch.
+* `subnet_type`: The type of the subnet.
+* `default_gateway_ip`: Default gateway IP address.
+* `prefix_length`: -. IP prefix length of the Subnet.
+* `subnet_ip`: Subnet IP address.
+* `dhcp_server_address`: Host address.
+* `dhcp_server_address_port`: Port Number.
+* `dhcp_options`: Spec for defining DHCP options.
+* `dhcp_domain_search_list`: -.
+* `vlan_id`: -.
```
Comment (yannickstruyf3) [inline website/docs/d/subnets.html.markdown]: VLAN assigned to the subnet

File: website/docs/d/subnets.html.markdown [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+* `project_reference`: The reference to a project.
+* `vswitch_name`: The name of the vswitch.
+* `subnet_type`: The type of the subnet.
+* `default_gateway_ip`: Default gateway IP address.
+* `prefix_length`: -. IP prefix length of the Subnet.
+* `subnet_ip`: Subnet IP address.
+* `dhcp_server_address`: Host address.
+* `dhcp_server_address_port`: Port Number.
+* `dhcp_options`: Spec for defining DHCP options.
+* `dhcp_domain_search_list`: -.
```
Comment (yannickstruyf3) [inline website/docs/d/subnets.html.markdown]: DHCP domain search list for a subnet

File: website/docs/d/subnets.html.markdown [hunk 13]
```diff
@@ -0,0 +1,85 @@
...
+* `value`: value of the key.
+
+### Reference
+
+The `project_reference`, `owner_reference`, `availability_zone_reference`, `cluster_reference`, `network_function_chain_reference`, `subnet_reference`.
+
+attributes supports the following:
+
+* `kind`: The kind name (Default value: project.
+* `name`: the nam.
```
Comment (yannickstruyf3) [inline website/docs/d/subnets.html.markdown]: name

File: website/docs/d/subnets.html.markdown [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+* `uuid`: the UUI.
```
Comment (yannickstruyf3) [inline website/docs/d/subnets.html.markdown]: UUID

**Review by yannickstruyf3 — CHANGES_REQUESTED** [review]

**Review by yannickstruyf3 — APPROVED** [review]

**Pull request merged**